SeaPublicWebsite/Views/EnergyEfficiency/ActionPlan/Partials/ReturnToActionPlanPartial.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@using SeaPublicWebsite.Models.EnergyEfficiency
+@model SeaPublicWebsite.Models.EnergyEfficiency.YourSavedRecommendationsViewModel;
+
+<h2 class="govuk-heading-m">
+    Returning to your Action Plan
+</h2>
+
+<div class="sea-user-reference govuk-!-margin-bottom-6" role="complementary">
+    <h2 class="govuk-body-l govuk-!-margin-0">Your reference code</h2>
+    <span class="govuk-heading-l" id="subsection-title">@Model.PropertyData.Reference</span>
+</div>
+
+<p class="govuk-body">
+    You can come back to this action plan using your reference number or we can send  a link to your email.
+</p>
+
+@if (Model.EmailAddress is not null)
+{
+    <p class="govuk-body sea-email-sent">
+        Email sent to @Model.EmailAddress
+    </p>
+}
+
+@{ await Html.RenderPartialAsync("SendEmailPartial.cshtml", new YourSavedRecommendationsEmailViewModel()); }
```

Comment:
Allows me to very nicely use only a small view model for the email button, instead of the entire model from this file.